python/redmonster/_version.py

```diff
@@ -1 +1 @@
-__version__ = '1.1.1.dev3'
+__version__ = 'v1.2.0'
```

Comment:
I don't think `__version__` is supposed to contain a 'v'.  It's fine if the *tag* does, but not necessarily the version in the code.